iGCS/UserInterface/ViewControllers/DefaultSettingsTableViewController.m

```diff
@@ -0,0 +1,234 @@
+            settingsWaypointCell.customTextField.tag = 0;
+
+        } else if ([cellContent isEqual:@"Ceiling"]) {
+            settingsWaypointCell.customTextField.text = [NSString stringWithFormat:@"%.2f",[GCSDataManager sharedInstance].gcsSettings.ceiling];
+            settingsWaypointCell.customTextField.tag = 2;
+
+        } else if ([cellContent isEqual:@"Radius"]) {
+            settingsWaypointCell.customTextField.text = [NSString stringWithFormat:@"%.2f",[GCSDataManager sharedInstance].gcsSettings.radius];
+            settingsWaypointCell.customTextField.tag = 1;
+        }
+        
+        cell.selectionStyle = UITableViewCellSelectionStyleNone;
+        [settingsWaypointCell.customTextField addTarget:self action:@selector(textFieldChanged:) forControlEvents:UIControlEventEditingDidEnd];
+        cell = settingsWaypointCell;
+
+    } else {
+        cell = [tableView dequeueReusableCellWithIdentifier:cellIdentifier];
+        if (cell == nil) {
+            cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:cellIdentifier];
+            
+            cell.textLabel.text = cellContent;
+            cell.selectionStyle = UITableViewCellSelectionStyleNone;
+            
+            if ([cellContent isEqual:@"Units"]) {
+                SettingsSegementedControlCell *settingsSegementedControlCell = [[SettingsSegementedControlCell alloc] initWithFrame:CGRectZero];
+                settingsSegementedControlCell.tableView = tableView;
+                cell.accessoryView = settingsSegementedControlCell;
+                
+            } else {
+                cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
+                cell.selectionStyle = UITableViewCellSelectionStyleDefault;
+            }
+        }
+    }
+    return cell;
+}
+
+-(NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section {
+    return [self.sectionKeysArray objectAtIndex:section];
+}
+
+
+#pragma mark - Table view delegate
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+
+    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+    NSString *key = [self.sectionKeysArray objectAtIndex:indexPath.section];
+    NSArray *contents = [self.sectionContentsDict objectForKey:key];
+    NSString *cellContent = [contents objectAtIndex:indexPath.row];
+    
+    if ([cellContent isEqual: @"Contact Us"]) {
+        NSURL *contactInfo = [NSURL URLWithString:[NSString stringWithFormat:@"mailto:contact@fightingwalrus.com?subject=iDroneCtrl%%20%@", [self getVersionInfo]]];
+        [[UIApplication sharedApplication] openURL:contactInfo];
+    }
+    
+    if ([cellContent isEqual: @"Order iDroneLink"]) {
+        [[UIApplication sharedApplication] openURL:[NSURL URLWithString:@"http://www.fightingwalrus.com/products/idronelink"]];
```

Comment:
Let's add ?ref=idronectrlapp to the end of the url so we can see how much traffic or sales this drives: `http://fightingwalrus.com/products/idronelink?ref=idronectrlapp`